shell/app/modules/project/types/issue.d.ts

```diff
@@ -237,3 +237,14 @@ declare namespace ISSUE {
   }
 }
 
+interface DrawerData {
+  [k: string]: any;
```

Comment:
this type can't be enumerate？

Reply:
May have these fields: labels, source, title, content, and custom fields.  The number of fields is not fixed

Reply:
u should list all of them once u can count it. If there are custom fields, should maintain in a separate field names `customFields`